docs/tutorials/intro.md

```diff
@@ -129,19 +125,15 @@ Users are also free to design their own providers that best fit into their ecosy
 
 `BufferProvider` and `AnyProvider` abstract over different ways the data may be loaded: `BufferProvider` abstracts over data providers that deserialize data, whereas `AnyProvider` abstracts over data providers that can provide concrete Rust objects, like data providers that use [`databake`](https://docs.rs/databake).
 
-In this tutorial we are going to use ICU4X's test data provider and then move on to a synchronous file-system data provider which uses ICU4X format JSON resource files.
-
-## Test data
-
-ICU4X's repository comes with pre-generated test data that covers all of its keys for a select set of locales. For production use it is recommended one use the steps in [Generating Data](#generating-data) to generate custom data, but for the purposes of learning how to use ICU4X, it is sufficient to use the data providers exported by `icu_testdata`. `icu_testdata::any()` will produce an `AnyProvider`, and `icu_testdata::buffer()` will produce a `BufferProvider`. The `any()` provider requires fewer features be enabled so we're using that.
+In this tutorial we are going to use ICU4X's test data provider and then move on to a generate our own data.
 
 # 6. Using an ICU4X component
 
-We're going to write an app that uses the `icu::datetime` component to format a date and time in a Japanese locale.
+We're going to extend our app to use the `icu::datetime` component to format a date and time.
 
-First, we need to register our choice of the provider in `./Cargo.toml`, which can be done by running:
+First, we need to register our choice of the provider, `icu_testdata`, in `Cargo.toml`, which can be done by running:
```

Comment:
issue: I think we should still explicitly mention that testdata is just test data we ship with for experimentation purposes